src/client/Microsoft.Identity.Client/OAuth2/MsalTokenResponse.cs

```diff
@@ -75,6 +78,18 @@ internal class MsalTokenResponse : OAuth2ResponseBase
         [JsonProperty(PropertyName = TokenResponseClaim.SpaCode)]
         public string SpaAuthCode { get; set; }
 
+        [JsonProperty(PropertyName = TokenResponseClaim.Authority)]
+        public string AuthorityUrl { get; set; }
```

Comment:
Can add a comment for these properties that they are only available from the Android broker?

Reply:
Yeah, you are right. I didn't realize we already have a mechanism for other answers here. This is a better solution than to modify the existing response.

Reply:
Added MobileTokenResponse